tests/main/dbus-activation-session-legacy/task.yaml

```diff
@@ -0,0 +1,52 @@
+summary: D-Bus session services support activation without a user of instance systemd
+
+prepare: |
+    snap set system experimental.user-daemons=true
+    snap set system experimental.dbus-activation=true
+
+restore: |
+    systemctl stop private-session-bus.service || true
+    rm -f dbus-launch.env run-session-bus.sh
+    snap unset system experimental.dbus-activation
+    snap unset system experimental.user-daemons
+
+execute: |
+    # This filters out Ubuntu 14.04 and Ubuntu Core respectively
+    if [ "$(command -v systemd-run)" == "" ] || [ "$(command -v dbus-launch)" == "" ]; then
```

Comment:
This was more an attempt at doing a feature test rather than enumerating which systems it would run on.  The idea being to not have to make decisions about future systems added to the test suite.